hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/hudi/functional/TestStreamingSource.scala

```diff
@@ -46,7 +46,7 @@ class TestStreamingSource extends StreamTest {
   )
   private val columns = Seq("id", "name", "price", "ts")
 
-  val handlingMode: HollowCommitHandling = BLOCK
+  val handlingMode: HollowCommitHandling = USE_TRANSITION_TIME
```

Comment:
Should this be removed now?  Or do we want to keep it for incremental pulls on 0.x tables?

Reply:
This should be removed